src/PMS/PMS.cpp

```diff
@@ -270,6 +270,41 @@ int PMSBase::pm25ToAQI(int pm02) {
     return 500;
 }
 
+/**
+ * @brief Correction PM2.5
+ * 
+ * @param pm25 Raw PM2.5 value
+ * @param humidity Humidity value (%)
+ * @return float 
+ */
+int PMSBase::pm25Compensated(int pm25, float humidity) {
+  float value;
```

Comment:
We need to link to the documentation where this is from

Reply:
@airgradienthq  please add your document for this formula.